AirCasting/Graph/Graph.swift

```diff
@@ -173,6 +198,7 @@ struct Graph: UIViewRepresentable {
     
     @ObservedObject var stream: MeasurementStreamEntity
     @ObservedObject var thresholds: SensorThreshold
+    var isSessionMobile: Bool
```

Comment:
Until now the `UI_PollutionGraph` had no reference to any of the business models which created a stable standalone component that could be used for other functionality. and the `Graph` was the one that glue it together. Imagine using `UI_PollutionGraph` in different place for displaying anything else. Despite the name of the class which already is a little wrong because it not only represents Pollution but also microphone session data it was  a nice component. Maybe the `UI_PollutionGraph` can expose a more generic method that could be used in the graph?

Reply:
Also maybe a name that represents what it does and not what it is used for `isSessionMobile` and what it relay doe is `zoomoutToThirtyMinutes`.

Reply:
I renamed this class to `AirCastingGraph`.

Reply:
still maybe not ideal with auto zoom enabled. In few weeks time You will not remember what it is autozooming to. It is a graph so what autozoom means exactly. But lets go with this name. Defiantly much better then isSessionMobile👍